kubedriver/kegd/model/deploy_helm_action.py

```diff
@@ -4,29 +4,53 @@ class DeployHelmAction:
 
     action_name = 'helm'
 
-    def __init__(self, chart, name, namespace=None, values=None, chart_encoded=False, tags=None, wait=None, timeout=None):
+    def __init__(self, chart, name, namespace=None, values=None, setfiles=None,
+                 chart_encoded=False, tags=None, wait=None, timeout=None):
         if chart is None:
             raise InvalidDeploymentStrategyError(f'{DeployHelmAction.action_name} action missing \'chart\' argument')
         if name is None:
             raise InvalidDeploymentStrategyError(f'{DeployHelmAction.action_name} action missing \'name\' argument')
         self.chart = chart
         self.name = name
         self.namespace = namespace
-        self.values = values
+
+        if values is not None:
+            if isinstance(values, str):
+                # Multiple value files are to be passed as a list of strings
+                # Keep this check for backwards compatibility with older assemblies.
+                self.values = [values]
+            elif not isinstance(values, list):
+                raise InvalidDeploymentStrategyError(f'{DeployHelmAction.action_name} expected values to be a list of strings, found '+str(isinstance(values))+' instead')
```

Comment:
`isinstance(value)` call in error should be `type(value)`

Reply:
Ah. will fix it